projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.ts

```diff
@@ -19,6 +19,8 @@ export class ReportBPComponent implements OnInit {
   submitted = false;
   min = 11;
   max = 250;
+  
+  @Output() resourceCreated = new EventEmitter();
```

Comment:
Please add the type param for this EventEmitter, otherwise it defaults to `any`.
If it is emitting the number of resources that were created, it should probably be `EventEmitter<number>`

Reply:
ok colin

Reply:
Okay, @semicolin I will add the type params for the EventEmitter.